src/scalability/horizontal-or-vertical.md

```diff
@@ -0,0 +1,15 @@
+# Horizontal or vertical
+
+Methods of adding more resources for a particular application fall into two broad categories: horizontal and vertical scaling.
+
+## Horizontal Scaling
+
+Horizontal scaling, which is also known as scaling out, means to add more machines to a system and distribute the load across multiple smaller machines. As computer prices have dropped and performance continues to increase, high-performance computing applications have adopted low-cost commodity systems for tasks. System architects may configure hundreds of small computers in a cluster to obtain aggregate computing power that often exceeds that of computers based on a single traditional processor. The development of high-performance interconnects such as [Gigabit Ethernet](https://en.wikipedia.org/wiki/Gigabit_Ethernet), [InfiniBand](https://en.wikipedia.org/wiki/InfiniBand) further fueled this model. 
```

Comment:
```suggestion
Horizontal scaling, which is also known as scaling out, means adding more machines to a system and distribute the load across multiple smaller machines. As computer prices have dropped and performance continues to increase, high-performance computing applications have adopted low-cost commodity systems for tasks. System architects may configure hundreds of small computers in a cluster to obtain aggregate computing power that often exceeds that of computers based on a single traditional processor. The development of high-performance interconnects such as [Gigabit Ethernet](https://en.wikipedia.org/wiki/Gigabit_Ethernet), [InfiniBand](https://en.wikipedia.org/wiki/InfiniBand) further fueled this model. 
```

Reply:
Note: 
- "mean to do" means "plan to do" or "intend to do"
- "mean doing" means "indicate something"

Reply:
@CaitinChen In your change, I think you missed `...and distributing the load...`

Reply:
@Hoverbear Thanks, dear bear!

Reply:
@Connor1996 I have updated my change.